cmd/config/internal/commands/cmdset_test.go

```diff
@@ -76,7 +76,7 @@ spec:
 		},
 		{
 			name: "set replicas no description",
-			args: []string{"replicas", "4"},
+			args: []string{"replicas", "--value", "4"},
```

Comment:
keep these tests as they are to test backward compatibility

Reply:
Done.